OJS.Workers.SubmissionProcessors/Models/OjsSubmission.cs

```diff
@@ -38,5 +38,7 @@ public string Code
         public string AllowedFileExtensions { get; set; }
 
         public TInput Input { get; set; }
+
+        public int MaxPoints { get; set; }
```

Comment:
This should be into the `TInput`, more specifically in the `TestsInputModel`, because simple input does not need `MaxPoints`, since it is not competition oriented.